packages/NodeTypeResolver/src/PerNodeTypeResolver/MethodCallTypeResolver.php

```diff
@@ -69,15 +70,18 @@ public function resolve(Node $methodCallNode): array
         return $this->resolveMethodReflectionReturnTypes($methodCallNode, $methodCallVariableType, $methodCallName);
     }
 
-    private function resolveMethodCallName(MethodCall $methodCallNode): ?string
+    private function resolveMethodCallName(MethodCall $methodCallNode): string
     {
         if ($methodCallNode->name instanceof Variable) {
             return (string) $methodCallNode->name->name;
         }
 
         if ($methodCallNode->name instanceof PropertyFetch) {
-            // not implemented yet
-            return null;
+            return (string) $methodCallNode->name->name;
+        }
+
+        if ($methodCallNode->name instanceof Class_) {
```

Comment:
@TomasVotruba Should we check the instance of `Class_` or we can generalize to `MagicConst`? As they all have the [`getName` method](https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Scalar/MagicConst.php#L27)?